sdk/tests/deqp/framework/opengl/gluDrawUtil.js

```diff
@@ -40,7 +40,7 @@ var gluShaderProgram = framework.opengl.gluShaderProgram;
  */
 gluDrawUtil.VertexArrayBinding = function(type, location, components, elements, data, stride, offset) {
     this.type = type;
-    this.location = location === undefined ? -1 : location;
+    this.location = location === undefined ? 0 : location;
```

Comment:
Why is this needed?  Who called with location undefined? From the caller side, gl.getAttribLocation() always returns a number.